docs/vagrant.md

```diff
@@ -7,25 +7,155 @@ You can use following guide to setup a cluster in a virtual machine.
 * Vagrant
 
 ### Preparation
-* bin/build-release.sh must rename _release/apache-storm-${RELEASE} to _release/storm-mesos-${RELEASE}
-* sudo echo "192.168.50.101 master" >> /etc/hosts
-* sudo echo "192.168.50.102 slave"  >> /etc/hosts
+1. Add name lookup entries for the 2 VMs:
+
+  ```shell
+  sudo sh -c 'echo "192.168.50.101 master" >> /etc/hosts'
+  sudo sh -c 'echo "192.168.50.102 slave"  >> /etc/hosts'
```

Comment:
For now, let's leave it be.  Easy to change later when the `agent` rename *actually* starts rolling out.